modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/service/SubscriptionDLAppHelperLocalServiceWrapper.java

```diff
@@ -180,6 +190,29 @@ private void _notifySubscribers(
 			if (Validator.isNotNull(friendlyURL)) {
 				entryURL = friendlyURL;
 			}
+			else {
+				FriendlyURLEntry friendlyURLEntry =
+					FriendlyURLEntryLocalServiceUtil.fetchMainFriendlyURLEntry(
+						_portal.getClassNameId(FileEntry.class.getName()),
+						fileVersion.getFileEntryId());
+
+				if (friendlyURLEntry != null) {
+					StringBundler sb = new StringBundler(4);
```

Comment:
This piece of code was supposed to be removed (since it is not needed) in https://github.com/liferay-lima/liferay-portal/pull/5015/commits/488ed935295e4d6cb96251c12afe729219b77da6.

I'll remove this commit at all to avoid noise (sorry)